hash/fmtowns_flop.xml

```diff
@@ -1762,6 +1832,40 @@ Zurukamashi Ver 2.0                                           Nikkonren Kikaku
 		</part>
 	</software>
 
+	<!-- Dumped from the original disks, no protection -->
+	<software name="shangrl">
+		<description>Shangrlia</description>
+		<year>1992</year>
+		<publisher>エルフ (Elf)</publisher>
+		<info name="alt_title" value="シャングリラ２　スペシャルディスク" />
```

Comment:
Is this correct?  That says Shangri-la 2 Special Disk, which appears to be the title of a different entry.  They don’t both have the same name in Japanese do they?

Reply:
That was a copy-paste mistake on my part, thanks for pointing it out. It's fixed now.